browser/src/control/Control.Zotero.js

```diff
@@ -319,6 +320,32 @@ L.Control.Zotero = L.Control.extend({
 		});
 	},
 
+	enableDialogFieldTypeCombobox: function(citationFormat) {
+		var entries = [];
+		if (citationFormat === 'note') {
+			entries = [_('Footnotes'), _('Endnotes')];
+		} else {
+			entries = [_('Fields'), _('Bookmarks')];
+		}
+		this.map.fire('jsdialogupdate', {
+			data: {
+				jsontype: 'dialog',
+				action: 'update',
+				id: 'ZoteroDialog',
+				control: {
+					id: 'zoterotype',
+					type: 'combobox',
+					entries: entries,
+					selectedCount: '1',
+					selectedEntries: [
+						this.getFieldType() === 'Bookmark' ? 1 : 0
```

Comment:
you introduced more field types, it that still correct?

Reply:
yes still correct :)